src/leagues/management/commands/setup.py

```diff
@@ -62,30 +54,28 @@ def scrape_associations(options):
         try:
             scrape_association(bhv_id, options)
         except Exception:
-            LOGGER.exception("Could not create Association")
+            logger.exception("Could not create Association")
 
 
-def scrape_association(bhv_id: int, options):
-    url = Association.build_source_url(bhv_id)
+def scrape_association(assocication_id: int, options):
+    url = Association.build_source_url(assocication_id)
     html = http.get_text(url)
     dom = parsing.html_dom(html)
-
     name = parsing.parse_association_name(dom)
-    try:
-        abbreviation = Association.get_association_abbreviation(name)
-    except KeyError:
-        LOGGER.warning("No abbreviation for association '%s'", name)
```

Comment:
I'm fine with restructuring this exception handling to a None-check.
I think the warning should still be logged though.